src/Cubie/Cube.hs

```diff
@@ -0,0 +1,10 @@
+module Cubie.Cube where
+
+import qualified Cubie.Corners as C
+import qualified Cubie.Edges as E
+
+data Cube = Cube { cornerOrientation :: C.OrientationVector
```

Comment:
I don't really want to nest the cube structure more, because pulling values out of the cube will become harder. With the cube just containing 4 vectors, we can pattern match and extract what we want easier, just like what you have above

Reply:
Okay cool that sounds good. What do you think about removing the record accessors since we'll use pattern matching anyway?